computeparticles/computeparticles.cpp

```diff
@@ -129,7 +117,7 @@ class VulkanExample : public VulkanExampleBase
 
 		VkClearValue clearValues[2];
 		clearValues[0].color = defaultClearColor;
-		clearValues[1].depthStencil = { 1.0f, 0 };
+		clearValues[1].depthStencil = { 1.f, 0 };
```

Comment:
This one is a matter of taste, and therefore falls in the "formatting change" category I mentioned earlier. It doesn't make the code objectively better, so it should be kept the way the original author wrote it.